packages/core/core/src/Parcel.js

```diff
@@ -387,6 +387,7 @@ export default class Parcel {
       },
       opts,
     );
+    return {unsubscribe: () => sub.unsubscribe()};
```

Comment:
The alternative to this change is making 

https://github.com/parcel-bundler/watcher/blob/52a2dd273568e196d1694bbfd8750643b8d62e5a/index.js.flow#L19-L21

an object type instead of an interface.